src/ServiceControl.Monitoring/Infrastructure/ReportThroughputHostedService.cs

```diff
@@ -27,7 +28,14 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
                     }
                     catch (Exception ex) when (ex is not OperationCanceledException)
                     {
-                        logger.LogError(ex, $"Error obtaining throughput from Monitoring for {ReportSendingIntervalInMinutes} minutes interval.");
+                        if (ex.InnerException is not null and QueueNotFoundException)
+                        {
+                            logger.LogError($"Error obtaining throughput from Monitoring for {ReportSendingIntervalInMinutes} minutes interval: {ex?.InnerException?.Message}");
+                        }
+                        else
+                        {
+                            logger.LogError(ex, $"Error obtaining throughput from Monitoring for {ReportSendingIntervalInMinutes} minutes interval.");
+                        }
```

Comment:
```suggestion
                        var exception = ex.InnerException is QueueNotFoundException qnfe ? qnfe : ex;

                        logger.LogError(exception, $"Error obtaining throughput from Monitoring for {ReportSendingIntervalInMinutes} minutes interval.");
```

Reply:
I want to display a different error and not log the whole exception if it's just a queue not found thing.